### PR TITLE
COSBETA-256 Do not show links to upload product images or formulation…

### PR DESCRIPTION
**Pull request by DWRendell**
… files if unable to edit notification

## Checklist:

- [ ] Automated checks are passing locally.
- [ ] CHANGELOG updated if change is worth telling users about.
### General testing
- [ ] Test without javascript
- [ ] Test on small screen
### Accessibility testing
- [ ] Works keyboard only
- [ ] Tested with one screen reader
- [ ] Zoom page to 400% - content still visible
- [ ] Disable css - does content make sense and appear in a logical order?
- [ ] Passes automated checker (automated in build or manual)

### CODE REVIEW

File: cosmetics-web/app/views/responsible_persons/notifications/show.html.slim [hunk 1]
```diff
@@ -11,4 +11,5 @@
   .govuk-grid-column-full
     = render "notifications/responsible_person_details", responsible_person: @notification.responsible_person
     = render "notifications/contact_person_details", contact_person: @responsible_person.contact_persons.first
-    = render "notifications/product_details", notification: @notification
+    = render "notifications/product_details", notification: @notification, \
+      allow_edits: !@notification.notification_complete?
```
Comment (tom-bell-softwire) [inline cosmetics-web/app/views/responsible_persons/notifications/show.html.slim]: I think `allow_edits` should always be `false` on this page.

File: cosmetics-web/app/views/responsible_persons/notifications/edit.html.slim [hunk 2]
```diff
@@ -14,7 +14,8 @@
   .govuk-grid-column-full
     = render "notifications/responsible_person_details", responsible_person: @notification.responsible_person
     = render "notifications/contact_person_details", contact_person: @responsible_person.contact_persons.first
-    = render "notifications/product_details", notification: @notification
+    = render "notifications/product_details", notification: @notification, \
+      allow_edits: !@notification.notification_complete?
```
Comment (tom-bell-softwire) [inline cosmetics-web/app/views/responsible_persons/notifications/edit.html.slim]: Should `allow_edits` always be `true` on this page?

File: cosmetics-web/app/views/responsible_persons/notifications/show.html.slim (hunk 1 above)
Comment (DWRendell) [reply cosmetics-web/app/views/responsible_persons/notifications/show.html.slim]: Ah good point, didn't think of that

File: cosmetics-web/app/views/responsible_persons/notifications/edit.html.slim (hunk 2 above)
Comment (DWRendell) [reply cosmetics-web/app/views/responsible_persons/notifications/edit.html.slim]: Done

**Pull request merged**